packages/xod-client-browser/src/containers/App.jsx

```diff
@@ -107,7 +107,7 @@ class App extends client.App {
     const { project } = this.props;
 
     const xodballJSON = XP.toXodball(project);
-    const xodballName = XP.getProjectName(project);
+    const xodballName = XP.getProjectName(project) || 'my-project';
```

Comment:
???

Reply:
Will rebase very soon and it will gone away :)